app/models/filters/hud_filter_base.rb

```diff
@@ -59,6 +59,28 @@ def effective_project_ids
       @effective_project_ids.uniq.reject(&:blank?)
     end
 
+    # Limit the effective project ids to only those with enrollments that overlap the report range
+    # OR
+    # Projects with operating dates overlapping the range.
+    # Looking for both of these will enable reporting on non-HMIS participating projects open during
+    # the report range, and will allow the data quality checks for enrollments that are open
+    # outside of the operating end dates
+    def effective_project_ids_during_range(effective_range)
+      @effective_project_ids_during_range ||= begin
+        ids_with_enrollments = GrdaWarehouse::Hud::Project.
+          where(id: effective_project_ids).
+          joins(:enrollments).
+          merge(GrdaWarehouse::Hud::Enrollemnt.open_during_range(effective_range)).
```

Comment:
```suggestion
          merge(GrdaWarehouse::Hud::Enrollment.open_during_range(effective_range)).
```

It doesn't look like this was caught by your tests - maybe worth adding test coverage? Not sure what kind of tests we have for Filter currently, but I thought we had some.

Reply:
Ooh, thanks.  Yeah, I'm testing this manually right now, but I'll see if we have any HUD Filter tests.  Even some simple unit tests here would be nice.